src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -149,9 +161,17 @@
 
 (schema/defn ^:always-validate get-jvm-metrics :- JvmMetricsV1
   []
-  (let [runtime-bean (ManagementFactory/getRuntimeMXBean)]
+  (let [runtime-bean (ManagementFactory/getRuntimeMXBean)
+        gc-beans (jmx/mbean-names "java.lang:name=*,type=GarbageCollector")]
     {:heap-memory (jmx/read "java.lang:type=Memory" :HeapMemoryUsage)
      :non-heap-memory (jmx/read "java.lang:type=Memory" :NonHeapMemoryUsage)
+     :file-descriptors (clojure.set/rename-keys
+                        (jmx/read "java.lang:type=OperatingSystem" [:OpenFileDescriptorCount :MaxFileDescriptorCount])
+                        {:OpenFileDescriptorCount :used :MaxFileDescriptorCount :max})
+     :gc-stats (map #(conj {:name (.getCanonicalName %)}
```

Comment:
Alright, I'll take a shot at `{"<collector name>" {:count :total-time-ms}}`.

Right now, the schema for the list is:

```
(def GcStatsV1
  [{:name schema/Str
    :count schema/Int
    :total-time-ms schema/Int}])
```

That would need to change to something like:

```
(def GcStatsV1
  {(schema/OneOrMore schema/Str) {:count schema/Int
                                   :total-time-ms schema/Int}})
```

What's the proper Schema definition for a repeated keys which follow a particular pattern?